docs/source/basic_tutorials/using_cli.md

```diff
@@ -0,0 +1,55 @@
+# Using TGI through CLI
+
+You can use CLI tools of TGI to download weights, serve and quantize models, or get information on serving parameters. 
+
+## Installing TGI for CLI
+
+To install TGI to use with CLI, you need to first clone the TGI repository, then inside the repository, run
```

Comment:
I would include the commands to do this (the git clone + the `make install` part.